database/migrations/2020_01_08_043027_create_blogs_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
```

Comment:
where is the usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=41579321 Rolling Back Migration

Reply:
Em roll back ở function down() của laravel tạo sẵn lúc em generate migrate

Reply:
![Screen Shot 2020-01-15 at 11 58 22 AM](https://user-images.githubusercontent.com/59555237/72406311-5ccc5480-378e-11ea-86ba-a647ec421a9b.png)

Reply:
Rolling Back Migration: Được sử dụng khi tạo sai migration và muốn sửa chúng, muốn quay lại tại thời điểm chạy migration lỗi đó. Khi chạy lệnh: php artisan migrate:rollback
, nó sẽ chạy function down() của tất cả migration